applications/FluidDynamicsApplication/tests/cpp_tests/test_stokes_two_fluid.cpp

```diff
@@ -20,6 +20,7 @@
 #include "includes/model_part.h"
```

Comment:
Did you also include the mass conservation term to the two fluids Stokes element?

Reply:
No, It was a mistake that now is solved. Done!